add solution

nila20 opened the pull request:
[DEMO LINK](https://nila20.github.io/layout_landing-page/)

anastasiiavorobiova reviewed (changes requested):
Great work! 
<img width="1145" alt="Screenshot 2023-12-25 at 14 26 18" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/73e8c9ad-1c62-46a9-bcb4-14617d196e82">
Consider fixing validation errors 
<img width="789" alt="Screenshot 2023-12-25 at 14 25 34" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/dc836312-7b8e-483f-a65d-9b935220097b">
Consider fixing elements that overlap here

Viktor-Kost reviewed (changes requested):
GJ!

To improve:
1. add transition to button hover effects
2. input font colors should be the same
<img width="596" alt="Screenshot 2023-12-25 at 19 45 37" src="https://github.com/mate-academy/layout_landing-page/assets/117587463/1f4990b5-8b20-4598-b562-5b32b57ef9a2">

3. if you familiar with js it'd be much better to trigger `Recommended` section animation only once

nila20 commented:
Im have not familiar with js yet
I have fixed transition on menu items and on buttons,
removed overlapping on the top of the menu, by putting cross always visible.

TarasHoliuk reviewed (approved):
Well done :heavy_check_mark:
